docs/pages/upcoming-releases.mdx

```diff
@@ -79,34 +79,20 @@ members in other Access Lists.
 
 ## Teleport Policy
 
-| Version | Date               |
-|---------|--------------------|
-| 1.24.0  | September 9, 2024  |
-
-### 1.24.0
-
-#### New UI
-
-Access Graph will include navigation updates to enhance the usability and
-scalability of the web UI.
-
-#### Crown Jewels
-
-Access Graph will allow marking the most critical resources as "Crown Jewels,"
-providing enhanced traceability and auditing capabilities for them.
+| Version | Date         |
+|---------|--------------|
+| 1.25.0  | October 2024 |
 
-#### SSH Key Scanning
+### 1.25.0
 
-Access Graph will support scanning of SSH keys on servers and client machines 
-to help detect unauthorized access paths.
+TODO(r0mant, jakule)
```

Comment:
@r0mant @jakule Can you guys add what's going to be in Policy 1.25 to this section?